src/v/jl/jls_writer/jls_writer.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#pragma once
+
+#include "base/likely.h"
+#include "base/seastarx.h"
+#include "bytes/iobuf.h"
+#include "jl/jls/jls.h"
+#include "model/fundamental.h"
+#include "model/record.h"
+
+#include <seastar/core/iostream.hh>
+
+namespace jl::jls_writer {
```

Comment:
if cluster/jumbo_log_stm writes messages into jumbo log, then what is a jumbo log writer here? i'm guessing jls is referring to something else? either way, it's a confusion i have that some light documentation would fix.